### PR TITLE
adding kin/2022-03

### PR DESCRIPTION
**Pull request by Burkifa23**
### Pull Request check-list

- [x] Have you run `run.sh` or `node deploy.js -b test -l [language] -q [quarter]`? No, however, it follows the file structure suggested.
- [x] Have you checked for semicolons in the titles?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

For example, `./run.sh -l en -q 2022-01` or `node deploy.js -b test -l en -q 2018-02`.

### Description of contribution

Reach out to me for any concerns.

### CODE REVIEW

**Comment by imasaru** [comment]
Thanks, Frank!
We could consider changing the language code to the [official](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?code_ID=236) `kin` to avoid conflict with Kikuyu, the language granted the `ki` code in [ISO 639](https://en.wikipedia.org/wiki/ISO_639). 

@alimness @VitalikL thoughts? :-)

**Comment by Burkifa23** [comment]
Okay, I'm done changing it from ki to kin.

**Comment by VitalikL** [comment]
Thanks @Burkifa23 ! As we are now official GC app, we need to confirm the source of the translation of the lesson. Is this an official translation of the lesson?

**Comment by Burkifa23** [comment]
> Thanks @Burkifa23 ! As we are now an official GC app, we need to confirm the source of the translation of the lesson. Is this an official translation of the lesson?

Yes, it is translated by the Rwandan SDA church which I scanned. I do understand your concerns sadly there is no digital version. I had tried checking from [Sabbath school personal ministries archives](https://www.sabbathschoolpersonalministries.org/archives) but I did not find it there except on a Facebook group that posts the lessons daily called [IBYIGISHO BY'ISHURI RYO KU ISABATO](https://www.facebook.com/groups/1188338784852006/) which may be the only way to vouch for kin/2022-03. If you want I can send you the scanned pdf file.

**Comment by VitalikL** [comment]
@Burkifa23 Great! Yes, if you can provide the PDF, just in case GC will have any questions / concerns. I will add the Bible translation and work on merging your PR. Thanks for contribution!

**Comment by Burkifa23** [comment]
In case you don't receive the file or someone else wants the file, You can reach it from [grosfi.ch/V9m7mLUgNAw](https://www.grosfichiers.com/V9m7mLUgNAw). I was amazed by the bible passage reference parser so adding the bible verses would be very wonderful and if you need further help I will be available.  Although, I don't know CoffeeScript I can help in other ways. Note to use the  Bibiliya Yera (BYSB) as the bible version in order to get the 66 books as other versions either have more books or have outdated translations.

**Comment by VitalikL** [comment]
Thanks @Burkifa23 ! Just to confirm is this https://en.wikipedia.org/wiki/Kinyarwanda language? Because it seems like we can use `rw` language code for that.

**Comment by Burkifa23** [comment]
> Thanks @Burkifa23 ! Just to confirm is this https://en.wikipedia.org/wiki/Kinyarwanda language? Because it seems like we can use `rw` language code for that.

@VitalikL  that is Kinyarwanda. We can use  [rw] according to ISO 639-1 or [kin] according to ISO 639-2. Do you want me to change to [rw] or remain on [kin]?

File: src/kin/2022-03/01/01.md
Comment (VitalikL) [inline src/kin/2022-03/01/01.md:5]: @Burkifa23 Please see en/2022-04/01/01.md to see how to properly format Sabbath day

File: src/kin/2022-03/01/02.md
Comment (VitalikL) [inline src/kin/2022-03/01/02.md:4]: Extra line here

File: src/kin/2022-03/01/02.md
Comment (VitalikL) [inline src/kin/2022-03/01/02.md:2]: Date has to be a properly formatted date, see English version

File: src/kin/2022-03/01/02.md
Comment (VitalikL) [inline src/kin/2022-03/01/02.md:25]: Here, questions should be separated by the new line.

**Review by VitalikL — CHANGES_REQUESTED** [review]

**Comment by VitalikL** [comment]
@Burkifa23 I am not sure if you were creating files manually, but they are not currently in a format that would work for the app. Especially dates, have to be in a proper format. You can use `create.js` script to generate the structure for the quarterly.

Additionally, please remove `package-lock.json` from the commits

**Comment by Burkifa23** [comment]
Hello VitalikL
I want to add that I am not a church employee but just a member. My
motivation to have this language posted is that it is spoken by almost a
million Adventists worldwide and it pains me that it is not found on the
application.

I want to understand the approval process before the language is posted on
the app.

My father consulted the head of the translation department at the union
level asking him if he could support us. But the answer was, the problem is
known by the Union.

What is the likelihood that my contributions will be taken into
consideration?

Do you have a time frame between the first submission and the posting on
the app?

Sincerely,
Frank.

**Comment by Burkifa23** [comment]
All previous mistakes were fixed. Week 1- 2 of 2022-04 is ready.

**Comment by Burkifa23** [comment]
Hello VitalikL,
I did create the files manually because I didn't know how to use
create.js.  I realized that  English dates are used even in other languages
so I will apply the same changes to even the fourth quarter which
am preparing.

**Comment by VitalikL** [comment]
Ooops! Looks like you have to fix some issues before I can merge this PR
||Error description |
| ----------- | ----------- |
|🛑| Critical error. Can not parse the quarterly info: `src/kin/2022-04`/info.yml. Error: `YAMLException: bad indentation of a mapping entry (19:16)<br><br> 16 |     - name: Ushinzwe Imirimo yo Gusohora Inyandiko<br> 17 |       value: "Lea Alexander Greve"<br> 18 |     - name: Ushinzwe Ubwanditsi Wungirije<br> 19 |       value: ""Sharon Thomas-Crews"<br>---------------------^`|

**Comment by Burkifa23** [comment]
@VitalikL Hi ! Hope you are doing well.
Where are the errors so that I can correct them, if possible?  I would appreciate it very much if you helped me with it.

**Comment by VitalikL** [comment]
Hi @Burkifa23. I was able to work on it on my end (Quarter 4, 2022. plus added the Bible so the links would be clickable). Can you please sync your fork and create Pull Request for new lessons (days) that are empty for this quarter. I also made formatting adjustments to the content for kin/2022-04 - please follow that format. And for now, I will close this PR. Thank you for your contribution

**Pull request closed without merging**
